cms/envs/production.py

```diff
@@ -254,10 +254,10 @@
                             service_variant=SERVICE_VARIANT)
 
 #theming start:
-PLATFORM_NAME = ENV_TOKENS.get('PLATFORM_NAME', PLATFORM_NAME)
-PLATFORM_DESCRIPTION = ENV_TOKENS.get('PLATFORM_DESCRIPTION', PLATFORM_DESCRIPTION)
-STUDIO_NAME = ENV_TOKENS.get('STUDIO_NAME', STUDIO_NAME)
-STUDIO_SHORT_NAME = ENV_TOKENS.get('STUDIO_SHORT_NAME', STUDIO_SHORT_NAME)
+PLATFORM_NAME = ENV_TOKENS.get('PLATFORM_NAME') or PLATFORM_NAME # Allow using default translatable name when the variable is missing or empty string
+PLATFORM_DESCRIPTION = ENV_TOKENS.get('PLATFORM_DESCRIPTION') or PLATFORM_DESCRIPTION # Allow using default translatable name when the variable is missing or empty string
+STUDIO_NAME = ENV_TOKENS.get('STUDIO_NAME') or STUDIO_NAME # Allow using default translatable name when the variable is missing or empty string
+STUDIO_SHORT_NAME = ENV_TOKENS.get('STUDIO_SHORT_NAME') or STUDIO_SHORT_NAME # Allow using default translatable name when the variable is missing or empty string
```

Comment:
This style of commenting makes the lines hard to read.  Put the comment once before the section.  But more importantly: is this the right pattern? Should we be using it more?  Why did we need to change the pattern here?

Reply:
Changed, please review it again. And as explained in the new comment, the reason is have the ability to activate **Translation** for the environment variable without deleting the line from **`lms.env.json`** and **`cms.env.json`** files. Instead; the admin should set the variable value as empty string **`""`** . With the original **default value** pattern; the admin must delete the variable from the **json** files

We need it more if we want to activate the translation for something else. Currently we have four environment variables that can be translated (but still need this PR to fix the json serialization but):
**`PLATFORM_NAME`**
**`PLATFORM_DESCRIPTION`**
**`STUDIO_NAME`**
**`STUDIO_SHORT_NAME`**